### PR TITLE
Add ubuntu mono theme

### PR DESCRIPTION
**Pull request by Kuameh**

### CODE REVIEW

**Review by Richienb — CHANGES_REQUESTED** [review]
Remember to update the readme with the new theme

**Comment by Kuameh** [comment]
Ok sure on it

File: README.md [hunk 1]
```diff
@@ -50,13 +50,13 @@ The `users` directory contains a list of files, each representing a host on mit-
...
-* gravatar
-* theme
-* license
+- copyright (required)
+- URL
+- email
+- format
+- gravatar
+- theme
+- license
```
Comment (Richienb) [inline README.md:59]: Don't make unrelated changed

Comment (Kuameh) [reply README.md:59]: I have a formator install, it made some changes on save.

Comment (Richienb) [reply README.md:59]: This still means you'll have to revert these changes.

**Pull request merged**